sdk/python/kfp_tekton/compiler/compiler.py

```diff
@@ -1001,6 +1002,7 @@ def _create_and_write_workflow(self,
         params_list,
         pipeline_conf)
     # Separate loop workflow from the main workflow
+    print(self.loops_pipeline)
```

Comment:
Why does kfp-tekton produce that to standard output?

Reply:
sorry it was a mistake, I removed it at https://github.com/kubeflow/kfp-tekton/pull/524